### PR TITLE
Adds tacos to food list, updates YAML suggestions

### PR DESCRIPTION
**Pull request by Happyman31**
This is a part of the git workshop!
I have added tacos to favourite foods and then suggested more YAML work in the NextTime.yaml file.

### CODE REVIEW

**Comment by tayloryoung6396** [comment]
The title should be something like "Adds tacos to food list, updates YAML suggestions". You also don't have to add your name

File: Foods.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias"]
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "tacos"]
```
Comment (andrewdeanphillips) [inline Foods.yaml]: Are tacos really that good though?

Comment (lach-j) [reply Foods.yaml]: Yes? :taco:

Comment (Happyman31) [reply Foods.yaml]: They can have whatever within them! From simple, too complicated!

**Review by andrewdeanphillips — CHANGES_REQUESTED** [review]
There needs to be more information on what is in the tacos.

File: NextTime.yaml [hunk 2]
```diff
@@ -19,4 +19,5 @@ suggestions:
   - Markup
   - CMake
   - WSL guide
+  - YAML work
```
Comment (tayloryoung6396) [inline NextTime.yaml:22]: What does YAML work mean? Like how YAML files are formatted/parsed? Or how we use them?

File: Foods.yaml [hunk 3]
```diff
@@ -6,7 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-
-
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "Tacos"]
-
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "tacos (mince & cheese"]
```
Comment (ysims) [inline Foods.yaml:9]:
```suggestion
  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "tacos (mince & cheese)"]
```
You forgot to close the parentheses :stuck_out_tongue:

**Pull request closed without merging**